src/astt_gui/templates/index.html

```diff
@@ -86,7 +86,7 @@
         <li class="nav-item dropdown">
 
           <a class="nav-link nav-icon" href="#" data-bs-toggle="dropdown">
-            <i class="bi bi-bell"></i><span>Alarms</span><span class="badge bg-danger badge-number">4</span>
+            <i class="bi bi-bell"></i><span>Alarms</span><span class="badge bg-danger badge-number">75k</span>
```

Comment:
@USilere What is the number 75k for if I many ask? I am a little bit confused about that.Please help advise

Reply:
I was testing something I will revert it to the 4 it was.